ui/raidboss/data/03-hw/dungeon/xelphatol.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+// Xelphatol
+[{
+  zoneId: ZoneId.Xelphatol,
+  timelineFile: 'xelphatol.txt',
+  timelineTriggers: [
+    {
+      id: 'Xelphatol Short Burst',
+      regex: /Short Burst/,
+      beforeSeconds: 4,
+      condition: Conditions.caresAboutMagical(),
+      suppressSeconds: 5, // Timelines jump sometimes, so let's not be noisy.
```

Comment:
I really need to just make all timeline triggers auto-suppress for 2-5 seconds by default.

Reply:
Likely so. The edge cases where we *would* want the noise haven't come up yet, to my knowledge.